content/sensu-go/6.1/observability-pipeline/observe-schedule/collect-metrics-with-checks.md

```diff
@@ -34,7 +34,7 @@ sensuctl check set-output-metric-handlers collect-metrics influx-db
 
 ### Supported output metric formats
 
-The output metric formats that Sensu currently supports for check output metric extraction are `nagios`, `influxdb`, `graphite`, and `opentsdb`.
+The output metric formats that Sensu currently supports for check output metric extraction are `nagios`, `influxdb`, `graphite`, `opentsdb`, and `prometheus`.
```

Comment:
I don't think it makes sense to code block words like "Nagios," "InfluxDB," etc. unless they are in the Sensu code form ex. `nagios_perfdata`, `influxdb_line` etc. I would update them to the code form _or_ remove the code blocks and properly capitalize the tools.

Reply:
Yeah, I thought that was strange. I'll update to use the formats themselves.